config/runtime.exs

```diff
@@ -16,6 +16,19 @@ image_tag = System.get_env("IMAGE_TAG")
 branch = System.get_env("BRANCH")
 commit = System.get_env("COMMIT")
 
+github_cert = System.get_env("GITHUB_CERT") || :notset
+github_app_id = System.get_env("GITHUB_APP_ID") || :no_app_id
+
+decoded_cert =
+  case github_cert do
+    :noset -> :notset
+    other -> Base.decode64!(other)
+  end
```

Comment:
What is `:noset`? Why not default to nil? I don't see `notset` or `noset` used anywhere.

Reply:
The intention was to leave a message for one to configure github, this can be achieved with nil as well will ammend